### DISCUSSION TITLE
Need Update Plugin Allure TestOps for Jira

### DISCUSSION THREAD

**Discussion by NekrasovAM** [allure-framework/allure2#2168] [TestOps]
Hi! Please update the [plugin ](https://marketplace.atlassian.com/apps/1223442?tab=support&hosting=datacenter)for Jira Data Center! Now the plugin can be installed to a maximum of version 9.7.2. 
We cannot install this plugin because our company uses a newer version of Jira DC 9.11.2.
We want to use all the capabilities of Allure TestOps, please help us in this matter! :)

**Comment by eroshenkoam** [comment]
Hi, @NekrasovAM

Indeed, it was set to version 9.7.2. We've now updated it to the latest version. Everything should be functioning properly now. Please let us know once you've had a chance to check, and thank you for bringing this to our attention.
